src/System.Management.Automation/FormatAndOutput/out-console/ConsoleLineOutput.cs

```diff
@@ -87,8 +87,15 @@ internal DisplayCellsPSHost(PSHostRawUserInterface rawUserInterface)
 
         internal override int Length(string str, int offset)
         {
```

Comment:
Are the changes to this method necessary to fix the issue?

Reply:
No. But it's a refactor to `Length` to reduce ops while `str` is empty.